Aliucord/src/main/java/com/aliucord/Http.java

```diff
@@ -108,6 +110,87 @@ public String toString() {
         }
     }
 
+    public static class MultiBuilder {
```

Comment:
Seeing as this class keeps open streams, it should implement closeable and properly close all streams in the close method